docs/content/en/docs/operator-manual/piped/configuration-reference.md

```diff
@@ -60,12 +59,12 @@ spec:
 | Field | Type | Description | Required |
 |-|-|-|-|
 | type | string | The repository type. Currently, HTTP and GIT are supported. Default is HTTP. | No |
-| name | string | The name of the Helm chart repository. Note that is not a Git repository but a [Helm chart repository](https://helm.sh/docs/topics/chart_repository/). | Yes |
-| address | string | The address to the Helm chart repository. | Yes |
+| name | string | The name of the Helm chart repository. Note that is not a Git repository but a [Helm chart repository](https://helm.sh/docs/topics/chart_repository/). | Yes if type is HTTP |
```

Comment:
Nice. Either is fine with me but I would love to unify how to write it when this field is different depending on the condition: https://pipecd.dev/docs-dev/user-guide/configuration-reference/#analysismetrics

Reply:
Yours looks simpler so I will be fine with this.